contracts/svm/programs/express_relay/src/lib.rs

```diff
@@ -438,31 +439,31 @@ pub struct SwapArgs {
 impl SwapArgs {
     pub fn convert_to_v2(&self, swap_platform_fee_bps: u64) -> SwapV2Args {
         SwapV2Args {
-            deadline: self.deadline,
-            amount_searcher: self.amount_searcher,
-            amount_user: self.amount_user,
-            fee_token: self.fee_token,
-            referral_fee_bps: self.referral_fee_bps,
-            swap_platform_fee_bps,
+            deadline:              self.deadline,
+            amount_searcher:       self.amount_searcher,
+            amount_user:           self.amount_user,
+            fee_token:             self.fee_token,
+            referral_fee_ppm:      u64::from(self.referral_fee_bps) * FEE_BPS_TO_PPM,
+            swap_platform_fee_ppm: swap_platform_fee_bps * FEE_BPS_TO_PPM,
         }
     }
 }
 
 #[derive(AnchorSerialize, AnchorDeserialize, Eq, PartialEq, Clone, Copy, Debug)]
 pub struct SwapV2Args {
-    // deadline as a unix timestamp in seconds
+    /// deadline as a unix timestamp in seconds
     pub deadline:              i64,
     pub amount_searcher:       u64,
     pub amount_user:           u64,
-    // The referral fee is specified in basis points
-    pub referral_fee_bps:      u16,
-    // Token in which the fees will be paid
+    /// The referral fee is specified in mill percentile
```

Comment:
what is a `mill percentile`

Reply:
1 millionth percentile, i thought thats what we wanted to raise it to, but i can divi down the numbers as well